src/widgets/helper/ChannelView.cpp

```diff
@@ -425,6 +425,15 @@ void ChannelView::initializeSignals()
             }
         });
 
+    this->signalHolder_.managedConnect(
+        getIApp()->getWindows()->deleteBuffersRequested, [&](Channel *channel) {
```

Comment:
nit: I'd prefer us to be explicit in what we capture
```suggestion
        getIApp()->getWindows()->deleteBuffersRequested, [this](Channel *channel) {
```